srcpkgs/oomox/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'oomox'
 pkgname=oomox
-version=1.6.2
-revision=2
-_droid_icon_ver=6dc4c92689ea2ce99534d6f461283efdf8ffd270
-_gnome_icon_ver=2f3d8dd3ce59e260803660f158a7a91d643518bf
-_oomoxify_ver=fe1b4279bc5e4d39116c718313bf68e167daaabc
-_materia_theme_ver=125048a8cba7352e69e6d0b7973ded01889e2949
-_oomox_gtk_theme_ver=a0787a875347f1387f390b0e72402842d431eb45
+version=1.7.0.2
+revision=1
+_droid_icon_ver=1.0.2
+_gnome_icon_ver=5.5.3
+_oomoxify_ver=1.0.0.1
+_materia_theme_ver=v20180519
```

Comment:
`_materia_theme_ver` should be `20180519` without the leading `v`. The `v` gets added later in `distfiles`, see below.

Reply:
is it better to use @actionless 's suggestion (and use the same format for the other files) or no?

Reply:
@actionless’s suggestion uses a current snapshot, but Void’s policy in general is to use releases. So I would use version 20180519 (and everything else as is, too).

Reply:
@newbluemoon  the problem what developer of materia theme is not around on github from may and latest revision have a bugfix required to work with oomox (and also one new feature):

https://github.com/nana-4/materia-theme/pulls